examples/00-basic/13-manage_licensing.py

```diff
@@ -0,0 +1,85 @@
+"""
+.. _ref_manage_licensing:
+
+Manage the DPF licensing logic via the server context
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This examples shows how to manage the licensing logic of a DPF server using a `ServerContext`.
+
+Preventing DPF from checking licenses out and blocking increments is possible
```

Comment:
You can prevent DPF from checking licenses out and blocking increments by using the Entry context.